sda-commons-server-kafka/src/main/java/org/sdase/commons/server/kafka/topicana/TopicConfigurationBuilder.java

```diff
@@ -1,7 +1,5 @@
 package org.sdase.commons.server.kafka.topicana;
 
-import com.github.ftrossbach.club_topicana.core.ExpectedTopicConfiguration;
-
 public class TopicConfigurationBuilder {
```

Comment:
Is this file copied? If yes, we should add the license header

Reply:
This file was created by us. No license needed.